scimaenaga.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir['{app,config,db,lib}/**/*', 'MIT-LICENSE', 'Rakefile', 'README.md']
 
-  s.required_ruby_version = '>= 2.5.9', '<= 3.3'
```

Comment:
Please add your problematic version (3.3.3?) to the test matrix after 3.3.0 at https://github.com/StudistCorporation/scimaenaga/blob/5d90e9e99a65ed60d457470ad510b51374ce3fc0/.github/workflows/test.yaml#L58